com.github.foldynl.QLog.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: com.github.foldynl.QLog
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-23.08'
+command: qlog
+rename-desktop-file: qlog.desktop
+rename-icon: qlog
+finish-args:
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --own-name=org.freedesktop.secrets
```

Comment:
Required for qtkeychain. Without this, QLog prints error on first launch and is not able to load/save credentials.